extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -290,9 +302,9 @@ public function testGetContext(
 		}
 
 		// Mock WikiFactoryHub
-		$wikiFactoryHubMock = $this->getMockBuilder('WikiFactoryHub')
+		$wikiFactoryHubMock = $this->getMockBuilder( 'WikiFactoryHub' )
 			->disableOriginalConstructor()
-			->setMethods( [ 'getCategoryId', 'getCategoryShort', 'getWikiVertical', 'getWikiCategoryNames' ])
+			->setMethods( [ 'getCategoryId', 'getCategoryShort', 'getWikiVertical', 'getWikiCategoryNames' ] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "getWikiCategoryNames" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)